### PR TITLE
Commit hotfixes

### PR DESCRIPTION
**Pull request by Tulun**

### CODE REVIEW

File: src/features/vote/CommitPhase.tsx [hunk 1]
```diff
@@ -184,9 +184,11 @@ const CommitPhase: FC<Props> = ({
       const identifiers = Object.keys(watchAllFields);
       const values = Object.values(watchAllFields);
       for (let i = 0; i < identifiers.length; i++) {
-        if (values[i] !== "") {
+        if (values[i] !== "" && values[i] !== undefined) {
+          const idenSplit = identifiers[i].split("~");
+
           const val = {
-            identifier: identifiers[i],
+            identifier: `${idenSplit[0]} -- ${idenSplit[1]}`,
```
Comment (mrice32) [inline src/features/vote/CommitPhase.tsx:191]: Can you explain what this is doing?

Comment (Tulun) [reply src/features/vote/CommitPhase.tsx:191]: Purely visual. This is to show the identifier and timestamp from the name of the input.

It's a way of abstracting the data without doing a look up in the array of values.

Comment (Tulun) [reply src/features/vote/CommitPhase.tsx:191]: Going to confirm some stuff though.

Comment (mrice32) [reply src/features/vote/CommitPhase.tsx:191]: I see. Thanks!

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by daywiss — APPROVED** [review]
going to approve, but there is a non critical issue on confirm modal 
image

**Pull request merged**